test/pageobjects/products.page.ts

```diff
@@ -0,0 +1,75 @@
+import { $ } from "@wdio/globals";
+import Page from "./page.js";
+
+class ProductsPage extends Page {
+
+  public get servisesDropDowns() {
+    return $$('div[data-testid="rightArrow"]');
+  }
+  public get filterForm() {
+    return $('div[data-testid="filterForm"]');
+  }
+  public get cardUnit() {
+    return $('div[data-testid="cardWrapper"]');
+  }
+
+  public get equipmentCategoryLabels() {
+    return $$('img[data-testid="firstCategoryImage"]');
+  }
+  public get equipmentCategorySecondLevelLabels() {
+    return $$('img[data-testid="secondCategoryImage"]');
+  }
+  public get activeEquipmentLabel() {
+    return $("//label[contains(concat(' ',normalize-space(@class),' '),'active_label')]");
+  }
+
```

Comment:
@Anastas11aZakharova The same thing about the empty spaces.

Reply:
Fixed in the latest commit